diff -Nur powersoftplus-0.1.8.orig/libftdi/Makefile.in powersoftplus-0.1.8.chng/libftdi/Makefile.in
--- powersoftplus-0.1.8.orig/libftdi/Makefile.in	2006-11-22 14:34:03.000000000 +0100
+++ powersoftplus-0.1.8.chng/libftdi/Makefile.in	2008-02-04 09:59:59.000000000 +0100
@@ -9,13 +9,8 @@
 TABFILEDST = libd2xx_table.so
 
 all:
-	$(INSTALLCMD) -m 0644 $(LIBFILE) $(LIBUPATH)/$(LIBFILE)
-	$(INSTALLCMD) -m 0644 $(TABFILESRC) $(LIBUPATH)/$(TABFILEDST)
-	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBUPATH)/$(LIBSFILE)
-	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBPATH)/$(LIBSFILE)
-	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBUPATH)/$(LIBS0FILE)
-	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBPATH)/$(LIBS0FILE)
-	/sbin/ldconfig
+	ln -s -f $(LIBFILE) $(LIBSFILE)
+	ln -s -f $(LIBFILE) $(LIBS0FILE)
 clean:
 distclean:
 
@@ -26,8 +21,6 @@
 	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBPATH)/$(LIBSFILE)
 	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBUPATH)/$(LIBS0FILE)
 	ln -s -f $(LIBUPATH)/$(LIBFILE) $(LIBPATH)/$(LIBS0FILE)
-	/sbin/ldconfig
-	./runacfg
 
 uninstall:
 	rm -f $(LIBUPATH)/$(LIBFILE)
diff -Nur powersoftplus-0.1.8.orig/scripts/Makefile.in powersoftplus-0.1.8.chng/scripts/Makefile.in
--- powersoftplus-0.1.8.orig/scripts/Makefile.in	2006-11-22 14:50:23.000000000 +0100
+++ powersoftplus-0.1.8.chng/scripts/Makefile.in	2008-02-04 10:14:51.000000000 +0100
@@ -8,11 +8,6 @@
 distclean:
 
 install:
-	@for f in $(RCFILES) ; do \
-	    $(INSTALLCMD) -m 0755 $$f $(RCPATH)/$$f; \
-	done
-	/sbin/chkconfig --add powersoftplus
-	/sbin/chkconfig powersoftplus on
 
 uninstall:
 	@/sbin/chkconfig --del powersoftplus
